react/the_react_ecosystem/type_checking_with_proptypes.md

```diff
@@ -21,15 +21,15 @@ npm install --save prop-types
 
 Next, we want to import the PropTypes package in the component whose props we want to validate.
 
-```javascript
+```jsx
```

Comment:
This block doesn't need to be changed as it does not contain any JSX syntax that needs different highlighting.